src/steps/users/contact-field-equals.ts

```diff
@@ -1,81 +1,64 @@
 
-  /**
-   * Type of step (either Action or Validation).
-   */
   protected stepType: StepDefinition.Type = StepDefinition.Type.VALIDATION;
 
-  /**
-   * A string that can be evaluated as an ECMAScript-compatible regular expression. This is used to
-   * identify and evaluate this step in cucumber-like scenario files. You are encouraged to use
-   * named regex capturing groups that correspond to the expected fields below.
-   */
-  // tslint:disable-next-line:max-line-length
-  protected stepExpression: string = 'the (?<field>.+) field on JSON Placeholder user (?<email>.+) should (?<operator>be less than|be greater than|be|contain|not be|not contain) (?<expectedValue>.+)';
+  protected stepExpression: string = 'the (?<field>[a-zA-Z0-9_]+) field on iterable contact (?<email>.+) should (?<operator>be less than|be greater than|be|contain|not be|not contain) (?<expectedValue>.+)';
 
-  /**
-   * An array of Fields that this step expects to be passed via step data. The value of "field"
-   * will be used as the field value's key when passed over the step data Struct.
-   */
   protected expectedFields: Field[] = [{
     field: 'email',
     type: FieldDefinition.Type.EMAIL,
-    description: "User's email address",
-  }, {
+    description: 'the email address of the contact',
+  },
+  {
     field: 'field',
     type: FieldDefinition.Type.STRING,
-    description: 'Field name to check',
-  }, {
+    description: 'the API name of the field',
+  },
+  {
     field: 'operator',
     type: FieldDefinition.Type.STRING,
-    description: 'Check Logic (be, not be, contain, not contain, be greater than, or be less than)',
-  }, {
+    description: 'the operator to use when comparing expected/actual values (one of be less than, be greater than, be, contain, not be, or not contain)',
+  },
+  {
     field: 'expectedValue',
     type: FieldDefinition.Type.ANYSCALAR,
-    description: 'Expected field value',
+    description: 'field value that is expected',
   }];
 
   async executeStep(step: Step): Promise<RunStepResponse> {
     let apiRes: any;
     const stepData: any = step.getData().toJavaScript();
     const email: string = stepData.email;
     const field: string = stepData.field;
+    const operator: string = stepData.operator;
     const expectedValue: string = stepData.expectedValue;
-    const operator: string = stepData.operator.toLowerCase();
 
-    // Search JSON Placeholder API for user with given email.
     try {
-      apiRes = await this.client.getUserByEmail(email);
+      apiRes = await this.client.getContactByEmail(email);
     } catch (e) {
-      return this.error('There was a problem connecting to JSON Placeholder.');
+      return this.error('There was a problem checking the Contact: %s', [e.toString()]);
     }
 
     try {
-      if (apiRes.body.length === 0) {
+      if (!apiRes.user) {
         // If no results were found, return an error.
         return this.error('No user found for email %s', [email]);
```

Comment:
- [ ] Nit: `contact` instead of `user`.

Reply:
![image](https://user-images.githubusercontent.com/54834859/72124431-afc59680-339f-11ea-9399-53bb625a353f.png) 

The property returned by the response is a `user` so i don't think contact would work

Reply:
upon trying it. `contact` is `undefined`

Reply:
Oh, sorry!  I meant in the error message.